<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Filter</class>
 <widget class="QWidget" name="Filter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="porog">
       <property name="text">
        <string>Пороговая фильтрация, значение порога:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="porogValue">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>254</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="ocu">
     <property name="text">
      <string>Пороговая фильтрация методом Оцу</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QRadioButton" name="median">
       <property name="text">
        <string>Медианный фильтр, радиус ядра:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="medYadro">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="gaus">
       <property name="text">
        <string>Фильтр гауса, параметр σ:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="gausYadro">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QRadioButton" name="lineraMed">
       <property name="text">
        <string>Линейный усредняющий фильтр, радиус ядра:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="linYadro">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="sobel">
     <property name="text">
      <string>Фильтр собеля</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QRadioButton" name="CAS">
       <property name="text">
        <string>Contrast Adaptive Sharpnes, sharpness:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sharpness">
       <property name="maximum">
        <double>1.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="show">
       <property name="text">
        <string>Посмотреть</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="sawe">
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
